views/templates/admin/_partials/configuration.tpl

```diff
@@ -209,8 +209,8 @@
 							<thead>
 								<th class="col-md-1">{l s='Enable' mod='stripe_official'}</th>
 								<th class="col-md-2">{l s='Payment method' mod='stripe_official'}</th>
-								<th class="col-md-6">{l s='Relevant countries' mod='stripe_official'}</th>
-								<th class="col-md-3">{l s='Require activation' mod='stripe_official'} *</th>
+                <th class="col-md-6">{l s='Relevant countries' mod='stripe_official'}&nbsp;<b>*</b></th>
```

Comment:
`<strong> ` should be more appropriate, shouldn't it ?

Reply:
strong tag is use for keyword, in this case i think b tag is appropriate